### ISSUE TITLE
[FEATURE] Audio Events Agent (PANNs AudioSet Classifier)

### ISSUE THREAD

**Issue by scottchronicity** [scottchronicity/orpheus#112]
### Context

Orpheus classifies audio two ways today: BirdNET (species) and crow-tools (call type / age / intent for corvids). Everything else in the soundscape — dogs, humans, sirens, vehicles, power tools, footsteps — is invisible except for a handful of labels BirdNET happens to include and that the UI filters out via the hardcoded `NON_BIRD_SOUNDS` set in `services/orpheus_ui/backend/src/orpheus_ui/api/diagnostics.py`.

This issue adds a parallel audio-event classifier agent that runs on the same raw audio clips as BirdNET, producing AudioSet-style labels (527 classes). Results land in SQLite and flow through the event correlator. A new "Audio Events" page in the dashboard visualises them, reusing the filter / pagination / chart patterns already built for Birds and Crows.

**Outcome:** a real noise channel answering "what else is going on in the soundscape?"

### Architecturally Significant Requirements (ASRs)

**Interface (Contract):**
- New agent `agents/orpheus-agent-audio-events/` cloned from the `orpheus-agent-crow-detection` template.
- MQTT subscribe: `orpheus/audio/motion/events` (same raw-clip topic BirdNET consumes — runs in parallel, not downstream).
- MQTT publish: `orpheus/detection/audio/events`.
- One `Detection(detection_type="audio.classified")` published per AudioSet label above confidence threshold. Top label populated as `species_common` / `species_code="audioset_<m_id>"` so the existing Entity correlator keys work unchanged.
- New backend endpoint `GET /api/data/audio-events/history` mirroring the bird-history contract (date range, time-of-day window, labels CSV, page / page_size).
- New frontend page at `/audio-events` reusing `SpeciesFilter` (relabelled "Labels"), `DateRangeFilter`, and the existing chart components.

**Implementation (Internal Logic):**
- Model: PANNs MobileNetV2 (~25 MB, PyTorch native, 527 AudioSet classes). Fits the existing PyTorch stack without adding TensorFlow. `panns_inference` PyPI package drives inference.
- Model weights distributed via Git LFS in `artifacts/models/panns_mobilenetv2.pth` (matches the pattern for AVES / mt_70.pt / birdnet.onnx). `make download-models` provides an HF fallback.
- Confidence threshold ≥ 0.5 before publishing (PANNs emits probabilities over all 527 classes per clip; without a threshold we'd flood the DB).
- No label blocklist in the agent — all labels above threshold are forwarded. The correlator alias map (see `[CORE] Correlator Alias Map for Cross-Classifier Merging`) is the correct place to de-dupe cross-classifier signals.
- Entities page description broadens from "correlated animal detection events" to "correlated animal and audio events". Existing species filter picks up audio labels via the `all_species` response field with no extra wiring.

### Architectural Constraints

- Must run on Jetson Orin NX alongside BirdNET + crow-tools without saturating compute. Profile on Jetson before merge; if PANNs inference exceeds ~500 ms per clip, consider gating on audio-motion peak energy or downsampling input from 48 kHz to 32 kHz (PANNs's native rate).
- Must not replace the hardcoded `NON_BIRD_SOUNDS` filter in this PR — the Birds page keeps its current filter.
- PANNs and AudioSet are MIT / CC-BY — safe for open-source. We distribute the trained model, not the dataset.
- Model weights addition must use Git LFS (already configured for `*.pth`).
- Per-OS quickstart docs (Mac / Linux / Jetson / Windows) must be updated to mention the new agent in "What Gets Installed".
- Must be Python 3.9 compatible.

### Acceptance Criteria

```gherkin
Feature: Audio Events Agent

  Scenario: Classify a dog-barking clip
    Given the audio-events agent is running and subscribed to orpheus/audio/motion/events
    When an audio.motion event is published for a clip containing dog barks
    Then the agent publishes at least one Detection with detection_type="audio.classified" and species_common="Dog"
    And the detection persists to SQLite with the correct timestamp and clip path

  Scenario: Confidence threshold drops low-signal labels
    Given PANNs emits probabilities for all 527 classes on a clip
    When the highest probability is 0.3
    Then zero detections are published for that clip

  Scenario: Audio Events page renders aggregated data
    Given 100 audio.classified detections spanning the last 7 days
    When a user opens /audio-events with date range = 7 days
    Then the stat cards show the total count and unique-label count
    And the label distribution pie chart renders the top labels
    And the Labels filter dropdown lists every label present in the range
```

### Definition of Done

- [ ] New `agents/orpheus-agent-audio-events/` created by cloning `orpheus-agent-crow-detection` (MQTT sub/pub, classifier wrapper, pydantic config, systemd unit, pytest fixtures, Makefile).
- [ ] `panns_inference` wired into the classifier wrapper.
- [ ] PANNs MobileNetV2 weights committed to `artifacts/models/panns_mobilenetv2.pth` via Git LFS.
- [ ] Agent unit tests: confidence threshold, handles missing clip gracefully, publishes correct detection_type.
- [ ] New `/api/data/audio-events/history` endpoint with the pagination + labels-filter contract (mirrors bird-history). Backend tests mirror `TestBirdHistoryEndpoint`.
- [ ] New `services/orpheus_ui/frontend/src/pages/AudioEvents.tsx` + route + nav entry in Layout. Frontend test covers the page renders and the labels filter populates.
- [ ] Entities page description updated to "correlated animal and audio events from multiple sensors".
- [ ] Per-OS quickstart docs mention the new agent.
- [ ] Jetson profile captured in a comment on this issue (inference time per clip) before merge.

### Follow-ups (tracked here as bullets, not separate issues)

- **Jetson performance profile**: if three parallel classifiers saturate compute, gate PANNs on audio-motion peak energy or downsample.
- **NON_BIRD_SOUNDS deprecation**: once audio-events is producing real labels, retire the hardcoded filter in `diagnostics.py` in favor of filtering by `detection_type` only.
